### DISCUSSION TITLE
How to deploy the NequIP model for LAMMPS based on the last model  instead of the best model?

### DISCUSSION THREAD

**Discussion by hityingph** [mir-group/nequip#365] [General]
Now I am using the trick of renaming the last_model.pth as best_model.pth, and then building the NequIP model based on the last model using the command -"nequip-deploy build --train-dir". Therefore, I am wondering whether there is a more convenient way to build the NequIP model for LAMMPS based on the last model?

**Comment by Linux-cpp-lisp** [comment]
Hi @hityingph ,

Thanks for your interest in our codes!

This is definitely an oversight and I've added it to my list to add a `--checkpoint` option to `nequip-deploy` to simplify this.  Until then, your solution is likely the easiest one.

**Comment by Linux-cpp-lisp** [comment]
Hi @hityingph ,

I've added a `nequip-deploy build --checkpoint` flag in a new commit (https://github.com/mir-group/nequip/commit/4aabe9f4f6587fac8696eaacb50e2cbf9ff99dee) on `develop`;  please let me know if you find any issues with it.
